src/data/data.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "src": "https://global-uploads.webflow.com/63994dae1033718bee6949ce/649f3f286772ee18775d26f1_mby979eTPSDlRpr7H2MNSly1Sk-p-500.png",
+    "alt": "Drippi.ai",
+    "title": "Drippi.ai",
```

Comment:
we also need a link TO the tool, that would technically be the `src`, you can alternatively name it `url` or something